packages/transformer/src/test/TestUtils/TimelineTestUtil.ts

```diff
@@ -500,12 +504,23 @@ export async function runTimeline(
             accessToken,
             startChangeset: startIndex ? { index: startIndex } : undefined,
           });
+          expect(
+            expectThrow === false || expectThrow === undefined,
```

Comment:
I believe I would also have to change the .to.be.true to to.be.false to do this right? Except chai actually seems to only accept 'false' i.e. undefined doesn't mean false.

Reply:
wait sorry. I meant `!expectThrow`. confused myself :)

Reply:
I think i'd prefer to keep `expectThrow === false || expectThrow === undefined` because it feels easier to read to me than `!expectThrow`. Let me know if you still want me to change it and I will.